projects/plugins/jetpack/class.json-api.php

```diff
@@ -202,12 +202,17 @@ public function add( WPCOM_JSON_API_Endpoint $endpoint ) {
 	}
 
 	/**
-	 * Determine if a string is truthy.
+	 * Determine if a string is truthy. If it's not a string, which can happen with
+	 * not well-formed data coming from Jetpack sites, we still consider it a truthy value.
 	 *
 	 * @param string $value "1", "t", and "true" (case insensitive) are falsey, everything else isn't.
```

Comment:
Pre-existing issue, but since we're here
```suggestion
	 * @param string $value "1", "t", and "true" (case insensitive) are truthy, everything else isn't.
```